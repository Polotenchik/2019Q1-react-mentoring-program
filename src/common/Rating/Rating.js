import React from 'react';

const Rating = props => (<span className="rating">{ props.value }</span>);

export default Rating;
