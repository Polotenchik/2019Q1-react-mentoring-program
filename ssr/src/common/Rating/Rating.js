import React from 'react';

export const Rating = props => (
    	<span className='rating'>
      		{ props.value }
    	</span>
  	);
