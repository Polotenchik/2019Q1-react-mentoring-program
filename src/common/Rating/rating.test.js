import React from 'react';
import { shallow } from 'enzyme';
import { Rating } from './Rating';

const mockNumber = '42';

describe('<Rating />', () => {
  describe('Rendering', () => {
    it('should render rating label', () => {
      const wrapper = shallow(<Rating value={ mockNumber }/>);
      expect(wrapper).toMatchSnapshot();
    });
    it('should render rating label with mock value', () => {
      const wrapper = shallow(<Rating />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
