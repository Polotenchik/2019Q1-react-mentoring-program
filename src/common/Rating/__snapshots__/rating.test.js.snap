// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rating /> Rendering should render rating label 1`] = `
<span
  className="rating"
>
  42
</span>
`;

exports[`<Rating /> Rendering should render rating label with mock value 1`] = `
<span
  className="rating"
/>
`;
